docs/serverless-demo.md

```diff
@@ -1,28 +1,26 @@
-# Welcome to the Apollo Showcase Demo
+# Welcome to the **{{ graph.name }}** API! 🎉
 
-Welcome to the **{{ graph.name }}** API! 🎉 You can follow along in this README to walk through a guided demo or just explore yourself.
+You can follow along in this README to walk through a guided demo or just explore around yourself.
 
-> If you're new to Apollo, learn more about GraphOS in [our docs](https://www.apollographql.com/docs/studio/).
+# So what did clicking "Try a Demo Graph" do?
 
-## So what did clicking that button do to get me here?
+Behind the scenes, we've set up GraphOS to work with two GraphQL APIs we have hosted for you:
 
-In creating this demo, we've set up GraphOS to work with two GraphQL APIs we have hosted for you:
-
-- _spotify_: Exposes the Spotify REST API
+- _spotify_: Exposes the [Spotify REST API](https://developer.spotify.com/documentation/web-api)
 - _playback_: Exposes subscriptions for playback functionality coming from the Spotify REST API including the associated `mutation` operations
 
-This includes provisioning a serverless instance of the [Apollo Router](https://www.apollographql.com/docs/graphos/routing/cloud) that can route traffic between our two GraphQL APIs 🚀
+GraphOS also provisioned a serverless instance of the [Apollo Router](https://www.apollographql.com/docs/graphos/routing/cloud) that can route traffic between our two GraphQL APIs 🚀
 
 # Demo Guide: Where should I start?
 
-> There are links in this demo that might navigate you around, it might be helpful to open two windows to keep this README up on one side of the screen (or right-click->Open in a new window)\_
+> There are links in this demo that might navigate you around to different pages of Studio. It might be helpful to open two windows to keep this README up on one side of the screen (or right-click -> Open in a new window).
 
 ## Running operations
 
-You need a way to query your graph. [GraphOS Explorer](https://www.apollographql.com/docs/graphos/explorer) is built to make this easy. You can try querying this graph using [Explorer]({{ graph.url.explorer }}). Try running your first operation:
+You need a way to query your graph and [GraphOS Explorer](https://www.apollographql.com/docs/graphos/explorer) is the perfect tool to help you! Try running your first few operations by clicking on the play icon beside each operation below:
 
 ```gql
-query RootWebsiteQuery($offset: Int, $limit: Int) {
+query RoolWebsiteQuery($offset: Int, $limit: Int) {
```

Comment:
What is `RoolWebsiteQuery`?

Reply:
OH was it supposed to be "Root"? I worked off the string in the Studio UI page.

Reply:
But "RootWebsite" is still not clear to me 😅

Reply:
That is supposed to be the query that is used to load the initial page data. Any recommendations on what we would name there?

Reply:
Since this graph kind of exists outside of the client app, maybe we just name it what it's querying for? `MyPlaylists`?